web/client/components/mapcontrols/scale/ScaleBox.jsx

```diff
@@ -37,7 +37,7 @@ var ScaleBox = React.createClass({
     },
     onComboChange(event) {
         var selectedZoomLvl = parseInt(event.nativeEvent.target.value, 10);
-        this.props.onChange(selectedZoomLvl, this.props.scales[selectedZoomLvl]);
```

Comment:
It triggers a warning 
    warning.js:45 Warning: Failed propType: Invalid prop `mapStateSource` of type `number` supplied to `OpenlayersMap`, expected `string`. Check the render method of `Connect(OpenlayersMap)`.
since that value is assigned to the action field `mapStateSource` of `map.js:changeZoomLevel`. Alternatively one could write another action with matching signature.